### DISCUSSION TITLE
Windows 11: Limit transmission to a single instance

### DISCUSSION THREAD

**Discussion by RalfReddings** [transmission/transmission#5669] [Q&A] [answered]
If a Transmission is minimised to the system tray (when setting "Show Transmission in the notification area" is enabled), running Transmission again from the start menu opens another instance.
![](https://i.imgur.com/KQG5Ak6.png)

There does not seem to be a means to bring up an existing window from the task tray (Other than clicking on the tray icon with the mouse).

In this link, [Raise present window if launched  #742 · transmission](https://github.com/transmission/transmission/pull/742), the devs mentioned this is intended behaviour to facilitate remote connections/workflows.

I get it but I am using Transmission entirely as a local, generic BitTorrent solution, I have no intention of ever using it as a remote interface.

Is there a way to simply force a "single instance mode"? or some kind of command line call that I can use to bring up an existing window?

I have spent the better part of yesterday and today trying to simply find a way bring an existing Transmission window up, here is one of the things I tried: [WIN 10/11: Can anyone else confirm if this library works on windows 11? : AutoHotkey](https://old.reddit.com/r/AutoHotkey/comments/14jhn4a/win_1011_can_anyone_else_confirm_if_this_library/)

Thanks for any help.

**Comment by Pentaphon** [comment, marked as answer]
> If a Transmission is minimised to the system tray (when setting "Show Transmission in the notification area" is enabled), running Transmission again from the start menu opens another instance.

This issue is being discussed here: https://github.com/transmission/transmission/issues/255
No fix yet.
